subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2174,15 +2175,17 @@ static int handle_request(struct zoap_packet *request,
 	return r;
 }
 
-int lwm2m_udp_sendto(struct net_pkt *pkt, const struct sockaddr *dst_addr)
+int lwm2m_udp_sendto(struct net_context *net_ctx, struct net_pkt *pkt,
+		     const struct sockaddr *dst_addr)
 {
 	return net_context_sendto(pkt, dst_addr, NET_SOCKADDR_MAX_SIZE,
 				  NULL, K_NO_WAIT, NULL, NULL);
 }
 
 void lwm2m_udp_receive(struct lwm2m_ctx *client_ctx, struct net_pkt *pkt,
 		       bool handle_separate_response,
-		       int (*udp_request_handler)(struct zoap_packet *,
+		       int (*udp_request_handler)(struct net_context *net_ctx,
```

Comment:
Will do.  I think early on I was heeding the checkpatch warning about no new typedefs :P